### PR TITLE
User servlet dev

### PR DESCRIPTION
**Pull request by jpaints3**
You can ignore the prettier commit. The only thing that is important is the previous commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ If you use Google Cloud Shell, these are already installed.
 
 1. Clone the repo.
 
-       git clone https://github.com/josueetcom/TeachMe.git
+   git clone https://github.com/josueetcom/TeachMe.git
 
 2. `cd` into the repo and setup pre-commit hooks by running:
 
-       npm install
+   npm install
```
Comment (josueetcom) [inline README.md]: Can you undo these changes?
I think from a clean state you can do `git checkout main README.md && git add README.md && git commit -m "revert README change" --no-verify`

File: src/main/java/com/google/sps/servlets/UserServlet.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+@WebServlet("/user")
```
Comment (josueetcom) [inline src/main/java/com/google/sps/servlets/UserServlet.java]: How about `/users`?

File: src/main/java/com/google/sps/servlets/UserServlet.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+public class UserServlet extends HttpServlet{
+
+    @Override
+    public void doPost(){ // create
```
Comment (josueetcom) [inline src/main/java/com/google/sps/servlets/UserServlet.java]: You're going to need spaces between the `)` and `{` in these methods

**Review by josueetcom — APPROVED** [review]
Some minor changes needed but otherwise good to go

**Review by JoseGrangerSalgado — APPROVED** [review]
LGTM
